### PR TITLE
Minor wording (ko)

### PR DESCRIPTION
**Pull request by marocchino**
@yous review please.

### CODE REVIEW

File: exercises/baby_steps/problem.ko.md [hunk 1]
```diff
@@ -27,6 +27,6 @@ $ node program.js 1 2 3
 
 또 `process.argv`의 모든 요소는 문자열이므로 숫자로 *바꿀* 필요가 있으니 조심하셔야 합니다. 이것은 속성 앞에 `+`를 붙이거나 속성을 `Number()`에 넘겨서 할 수 있습니다. 예: `+process.argv[2]`, `Number(process.argv[2])`
 
-`{appname} verify program.js`를 실행할 때 {appname}가 인자를 넣어 주므로 직접 넣을 필요는 없습니다. 확인없이 프로그램을 테스트하기 위해서는, `{appname} run program.js`를 사용해 실행할 수 있습니다. `run`을 사용하면, 각 연습문제에 맞춰 {appname}가 설정해 준 테스트 환경을 사용해 실행할 수 있습니다.
+`{appname} verify program.js`를 실행할 때 {appname}가 인자를 넣어 주므로 직접 넣을 필요는 없습니다. 확인없이 프로그램을 테스트하기 위해서는, `{appname} run program.js`를 사용해 실행할 수 있습니다. `run`을 사용하면, 각 연습 문제에 맞춰 {appname}가 설정해 준 테스트 환경을 사용해 실행할 수 있습니다.
```
Comment (yous) [inline exercises/baby_steps/problem.ko.md]: 확인없이 → 확인 없이

File: exercises/juggling_async/problem.ko.md [hunk 2]
```diff
@@ -9,6 +9,6 @@
 
 결과를 큐에 넣어 어떤 URL이 전체 내용을 반환했는지 추적해야 합니다. 전부 다 받았을 때에만, 콘솔에 내용을 출력할 수 있습니다.
 
-카운팅 콜백은 Node에서 async를 관리하는 기초적인 방법입니다. 직접 구현하기보다, [async](http://npm.im/async)나 [after](http://npm.im/after)같은 서드 파티 라이브러리를 사용하는 것이 더 간편할 수 있습니다. 하지만 연습문제이니, 외부 라이브러리 없이 한 번 해보세요.
+카운팅 콜백은 Node에서 async를 관리하는 기초적인 방법입니다. 직접 구현하기보다, [async](http://npm.im/async)나 [after](http://npm.im/after)같은 서드 파티 라이브러리를 사용하는 것이 더 간편할 수 있습니다. 하지만 연습 문제이니, 외부 라이브러리 없이 한 번 해보세요.
```
Comment (yous) [inline exercises/juggling_async/problem.ko.md]: after같은 → after 같은

**Comment by yous** [comment]
I'm done.

**Comment by marocchino** [comment]
Thank you!

**Pull request merged**